package.json

```diff
@@ -19,12 +19,12 @@
   },
   "repository": {
     "type": "git",
-    "url": "git://github.com/pimatic/homeduinojs.git"
+    "url": "git://github.com/ronaldsteen/homeduinojs-dst.git"
   },
-  "author": "Oliver Schneider <oliverschneider89+sweetpi@gmail.com>",
```

Comment:
> The "author" is one person. "contributors" is an array of people. 

https://docs.npmjs.com/files/package.json#people-fields-author-contributors

Could you please add your name to `contributors` instead?
Please leave the issue and main url as it wars.